pkg/cloud/gcp/actuators/machine/reconciler.go

```diff
@@ -175,13 +207,6 @@ func (r *Reconciler) reconcileMachineWithCloudState(failedCondition *v1beta1.GCP
 		r.machine.Status.Addresses = nodeAddresses
 		r.providerStatus.InstanceState = &freshInstance.Status
 		r.providerStatus.InstanceID = &freshInstance.Name
-		succeedCondition := v1beta1.GCPMachineProviderCondition{
-			Type:    v1beta1.MachineCreated,
-			Reason:  machineCreationSucceedReason,
-			Message: machineCreationSucceedMessage,
-			Status:  corev1.ConditionTrue,
-		}
-		r.providerStatus.Conditions = reconcileProviderConditions(r.providerStatus.Conditions, succeedCondition)
```

Comment:
Are we sure we want to remove this?

Reply:
It is set when Create is successful. How often can create successfully happen? It is also the new sentinel in exists - the cloud instance must exist and the machine itself must have the status condition. We could add an annotation, but that's just more of the same.